### PR TITLE
Updates the client to implement a list of characters and a add-character-mutation

### PR DESCRIPTION
**Pull request by Vassyli**

### CODE REVIEW

File: app/components/App.js [hunk 1]
```diff
@@ -9,18 +9,26 @@ import SessionRoute from './../routes/SessionRoute';
...
-var Title = React.createClass({
+function Title(props) {
+    return <h1>{props.val}</h1>;
+}
+
+function Connection(props) {
+    return <div>Connected to {props.name}</div>
+}
+
+/*var Title = React.createClass({
```
Comment (austenmc) [inline app/components/App.js]: Do we really need to keep this commented out code?

File: app/components/CharacterSelect.js [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+ * @returns {Function|String}
+ */
+function CreationButton(props) {
+    if (props.toggle) {
+        return (
+            <div className="daenerys-add" onClick={() => {props.onClick();}}>^</div>
+        );
+    } else {
+        return (
+            <div className="daenerys-add" onClick={() => {props.onClick();}}>+</div>
```
Comment (austenmc) [inline app/components/CharacterSelect.js]: I would clean this up with something like 
```
let content = props.toggle ? '^' : '+';
return (
  <div className="daenerys-add" onClick={() => {props.onClick();}}>{{content}}</div>
);
```

Not sure if that's the exact syntax for printing a variable's content.

File: app/components/CharacterSelect.js [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+                onFailure: (response) => {this.onCharacterCreationError(creationForm, characterName)},
+                onSuccess: (response) => {this.onCharacterCreationSuccess(creationForm, characterName)}
+            }
+        );
+
+        this.setState({submitted: true});
+    },
+
+    /**
+     * Event upon the character creation failes.
```
Comment (austenmc) [inline app/components/CharacterSelect.js]: nit: "fails"

**Review by austenmc — APPROVED** [review]

File: app/components/App.js (hunk 1 above)
Comment (Vassyli) [reply app/components/App.js]: No, we don't.

File: app/components/CharacterSelect.js (hunk 2 above)
Comment (Vassyli) [reply app/components/CharacterSelect.js]: It's `const content` and `{content}`, but yes, you're right.

File: app/components/CharacterSelect.js (hunk 3 above)
Comment (Vassyli) [reply app/components/CharacterSelect.js]: fixed.

**Pull request merged**